### ISSUE TITLE
Admin: JAWS No confirmation of update on Season Dates

### ISSUE THREAD

**Issue by JohnHewittCriterion** [nciinc/fs-permit-platform#605]
## Notes
Steps To Verify With JAWS:
 
1.	Log on to the application and move to the Season Dates page at the following URL: https://forest-service-trees-staging.app.cloud.gov/admin/christmas-trees/season-dates
2.	Complete the form to change season dates.
3.	Activate the "Update Season Dates" button. No discernable action is taken.
4.	Move to the top of the page and begin reading content. Find a confirmation statement "Season dates for Shoshone National Forest are now Sep 30, 2018 to Dec 31, 2019".
A method must be implemented to notify user immediately the update is successful.
 
Make interactive content more predictable. Unexpected changes of context can be so disorienting for users with visual disabilities or cognitive limitations that they are unable to use the content.
 
User found the same results in the Firefox browser.

Linkage: CTF-84
WCAG: 3.2.2 On Input 

## Acceptance criteria
- [x] focus shifts to success banner after submit
## Tasks
- [x] add focus on success alert so that screen reader reads it

## Definition of Done
- [x] Code unit / e2e tested and comply with QWASP
- [x] No new security vulnerabilities
- [x] Internal team code review
- [x] Documentation / readme.md / CircleCI updated
- [x] Docker updated
- [x] Compare finished design with mockup
- [ ] Usability tested
